src/wanderer/yellowray.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  canAdventure,
+  getLocationMonsters,
+  itemDropsArray,
+  Location,
+  print,
+  toMonster,
+} from "kolmafia";
+import { sumNumbers } from "libram";
+import { garboValue } from "../session";
+import { canWander, DraggableFight, UnlockableZones, WandererTarget } from "./lib";
+
+function averageYrValue(location: Location, debug = false) {
+  const monsters = Object.keys(getLocationMonsters(location))
+    .map((m) => toMonster(m))
+    .filter((m) => !["LUCKY", "ULTRARARE", "BOSS"].some((s) => m.attributes.includes(s)));
+
+  if (monsters.length === 0) {
+    return 0;
+  } else {
+    return (
+      sumNumbers(
+        monsters.map((m) => {
+          const items = itemDropsArray(m)
+            .filter((drop) => ["", "n"].includes(drop.type))
+            .map((drop) => garboValue(drop.drop));
+
+          if (debug) print(`${m}: ${items.join(",")}`);
+
+          return sumNumbers(items);
+        })
+      ) / monsters.length
+    );
+  }
+}
```

Comment:
I am probably merging https://github.com/Loathing-Associates-Scripting-Society/garbage-collector/pull/1113 before this, so just watch out for the changes